code/modules/antagonists/vampire/vampire_subclasses.dm

```diff
@@ -7,6 +7,9 @@
 	var/list/fully_powered_abilities
 	/// Whether or not a vampire heals more based on damage taken.
 	var/improved_rejuv_healing = FALSE
+	/// maximun number of thralls a vampire may have at a time. incremented as they grow stronger, up to a cap at full power.
+	var/thrall_cap = 1
+	var/full_power_overide = FALSE
```

Comment:
Is this a debug variable?

Reply:
well yes, but I feel like leaving it in could be useful for testing/adminbus in the future. I already added this to the PR description